db/schema.rb

```diff
@@ -248,5 +261,6 @@
   add_foreign_key "reports", "backgrounds"
   add_foreign_key "reports", "pipeline_outputs"
   add_foreign_key "taxon_counts", "pipeline_outputs"
+  add_foreign_key "taxon_summaries", "backgrounds"
```

Comment:
The taxon_zscores table is now unused, right?   No action suggested, just making sure I understand.

Reply:
No, taxon_summaries could really be called "background_taxon_summaries" -- it is used together with taxon_counts to generate taxon_zscores

Reply:
Got it thanks.

Reply:
should we call the table background_taxon_summaries? if not, we should at least document in in taxon_summary.rb it's exclusively for background.

Reply:
ok I'll add a comment